assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -208,9 +210,11 @@ table.wc-block-cart-items {
 				}
 			}
 			.wc-block-cart-item__total {
-				grid-column-start: 3;
+				grid-column: 2 / span 2;
 				grid-row-start: 2;
-				align-self: center;
+				align-self: end;
+				justify-self: end;
+				width: fit-content;
```

Comment:
Is the width needed here? I thought inheriting it from the grid would be enough.

Reply:
Nope. You are right. Removed.
Chrome also has a grid inspector. Super useful tools, thanks for sharing.
![image](https://user-images.githubusercontent.com/17271089/97313645-29fbd480-1867-11eb-8724-e2fb650e79fe.png)